v19.1/remove-nodes.md

```diff
@@ -1,26 +1,35 @@
 ---
 title: Decommission Nodes
-summary: Permanently remove one or more nodes from a cluster.
+summary: Remove one or more nodes from a cluster.
 toc: true
 ---
 
-This page shows you how to decommission and permanently remove one or more nodes from a CockroachDB cluster. You might do this, for example, when downsizing a cluster or reacting to hardware failures.
+This page shows you how to decommission one or more nodes. Decommissioning a node removes it from the CockroachDB cluster.
 
-For information about temporarily stopping a node (e.g., for planned maintenance), see [Stop a Node](stop-a-node.html).
+You might do this, for example, when downsizing a cluster or reacting to hardware failures.
+
+{{site.data.alerts.callout_info}}
+In other scenarios, such as when [upgrading your cluster's version of CockroachDB](upgrade-cockroach-version.html) or performing planned maintenance (e.g., upgrading system software), you will want to temporarily [stop the node](stop-a-node.html) and restart it later.
+{{site.data.alerts.end}}
 
 ## Overview
 
 ### How it works
 
-When you decommission a node, all range replicas on the node are transferred to other nodes.
+A node is considered to be decommissioned when it meets two criteria:
+
+1. The node has completed the decommissioning process.
+2. The node has been stopped and has not [updated its liveness record](cluster-setup-troubleshooting.html#node-liveness-issues) for 5 minutes.
+
+The decommissioning process transfers all range replicas on the node to other nodes. During and after this process, the node is considered "decommissioning" and continues to accept new SQL connections. Even without replicas, the node can still function as a gateway to route connections to relevant data. However, note that the [`/health?ready=1` monitoring endpoint](monitoring-and-alerting.html#health-ready-1) considers the node "unready" and returns a `503 Service Unavailable` status response code so load balancers stop directing traffic to the node. This behavior has been fixed in v20.1.
 
-During and after decommissioning, the node continues to accept new SQL connections. Even without replicas, the node can still function as a gateway to route connections to relevant data. However, note that the [`/health?ready=1` monitoring endpoint](monitoring-and-alerting.html#health-ready-1) considers the node "unready" and returns a `503 Service Unavailable` status response code so load balancers stop directing traffic to the node. This behavior has been fixed in v20.1.
+After all range replicas have been transferred, it's typical to stop the node via a process manager or orchestration tool, or by sending `SIGTERM` manually. You can also use the [`cockroach quit`](stop-a-node.html) command to stop the node. When stopped, the node is drained of in-flight SQL connections and new SQL connections are rejected. At this point, the node is decommissioned.
```

Comment:
Let's not mention `./cockroach quit` since we wish to get rid of it.

Reply:
>  At this point, the node is decommissioned.

You say earlier that it needs to meet two criteria above, but it will take five minutes until it meets the second.